docs/storage/client.rst

```diff
@@ -23,6 +23,15 @@ Install the ``google-cloud-storage`` library using ``pip``:
 Usage
 -----
 
+.. note::
+
+   Becuase the :class:`~google.cloud.storage.client.Client` uses the
+   third-party :mod:`requests` library by default, it should be safe to
+   share instances across threads.  In multiprocessing scenarious, best
+   practice is to create client instances *after*
+   :class:`multiprocessing.Pool` or :class:`multiprocessing.Process` invokes
+   :func:`os.fork`.
```

Comment:
We may want to add a test for this case, "should be safe" would concern me.

Reply:
Our thread-safety issues in the past were due to using `httplib2`.  While individual `requests.session.Session` objects aren't thread-safe, we don't use them directly.  The session pool comes from `urllib3`, which is supposed to be so.  @theacodes Can you confirm?

Reply:
Yeah, urllib3 is threadsafe and so is requests.

Reply:
Can we update the wording to "is thread-safe" or is that too strong?